python/python/glide/async_commands/core.py

```diff
@@ -740,6 +740,39 @@ async def lrange(self, key: str, start: int, end: int) -> List[str]:
             ),
         )
 
+    async def lindex(
+        self,
+        key: str,
+        index: int,
+    ) -> Optional[str]:
+        """
+        Returns the element at `index` in the list stored at `key`.
+
+        The index is zero-based, so 0 means the first element, 1 the second element and so on.
+        Negative indices can be used to designate elements starting at the tail of the list.
+        Here, -1 means the last element, -2 means the penultimate and so forth.
```

Comment:
```suggestion
        The index is zero-based, so `0` means the first element, `1` the second element and so on.
        Negative indices can be used to designate elements starting at the tail of the list.
        Here, `-1` means the last element, `-2` means the penultimate and so forth.
```
Do we need to emphasize numbers?